javascript/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@dmsnell/diff-match-patch",
+  "version": "1.0.0",
+  "description": "Efficient and human-readable text diffs with helpers",
+  "keywords": [
+    "diff",
+    "diff-match-patch",
+    "google-diff-match-patch"
+  ],
```

[Let's declare it a module](https://nodejs.org/api/packages.html#type):

```suggestion
  ],
  "type": "module",
```

---

I'm working on a PR to add types, maybe that would be a good time to make the switch to ESM.